sdk/client/src/blocking.rs

```diff
@@ -52,6 +52,11 @@ impl BlockingClient {
         }
     }
 
+    #[allow(dead_code)]
+    pub(crate) fn take_retry(&mut self) -> Retry {
+        mem::replace(&mut self.retry, Retry::none())
```

Comment:
what's the point of the mem::replace versus `self.retry = Retry::none()`

Reply:
`mem::replace` gives you the retry you're replacing : )